pkgs/by-name/sl/sleek-grub-theme/package.nix

```
@@ -0,0 +1,30 @@
+{pkgs, lib, ...}:
+pkgs.stdenv.mkDerivation rec {
+
+  pname = "sleek-grub-theme";
+  version = "unstable-2022-06-04";
+
+  src = pkgs.fetchFromGitHub ({
+    owner = "sandesh236";
+    repo = "sleek--themes";
+    rev = "981326a8e35985dc23f1b066fdbe66ff09df2371";
+    sha256 = "sha256-yD4JuoFGTXE/aI76EtP4rEWCc5UdFGi7Ojys6Yp8Z58=";
+  });
+
+  banner = "Grub Bootloader";
+
+  installPhase = ''
+    mkdir -p $out/
```

```suggestion
    runHook preInstall
    
    mkdir -p $out/
```
Similarly for `postInstall`.